hydrolib/core/io/mdu/models.py

```diff
@@ -342,18 +344,29 @@ class Output(INIBasedModel):
     classmapfile: Optional[Path] = Field(None, alias="ClassMapFile")
     waterlevelclasses: List[float] = Field([0.0], alias="WaterlevelClasses")
     waterdepthclasses: List[float] = Field([0.0], alias="WaterdepthClasses")
-    classmapinterval: float = Field(0.0, alias="ClassMapInterval")
-    waqinterval: float = Field(0.0, alias="WaqInterval")
-    statsinterval: float = Field(0.0, alias="StatsInterval")
+    classmapinterval: List[float] = Field([0.0], alias="ClassMapInterval")
+    waqinterval: List[float] = Field([0.0], alias="WaqInterval")
+    statsinterval: List[float] = Field([0.0], alias="StatsInterval")
     writebalancefile: bool = Field(False, alias="Writebalancefile")
-    timingsinterval: float = Field(0.0, alias="TimingsInterval")
+    timingsinterval: List[float] = Field([0.0], alias="TimingsInterval")
     richardsononoutput: bool = Field(True, alias="Richardsononoutput")
 
     _split_to_list = get_split_string_on_delimiter_validator(
         "waterlevelclasses",
         "waterdepthclasses",
         delimiter=";",
     )
+    _split_to_list2 = get_split_string_on_delimiter_validator(
+        "hisinterval",
+        "xlsinterval",
+        "mapinterval",
+        "rstinterval",
+        "classmapinterval",
+        "waqinterval",
+        "statsinterval",
+        "timingsinterval",
+        delimiter=" ",
```

Comment:
I believe setting the delimiter to " " will cause the validation to fail if the parts have multiple whitespaces between them.